tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -5402,7 +5402,9 @@ def globalReadGuardKBody(tP):
               # for each component in vector
               while r < numLoadVectorComp:
                 numElementsPerLoad = 1
-                if kernel["ProblemType"]["DataType%s"%tcDataType].isInt8() or kernel["ProblemType"]["DataType%s"%tcDataType].is8bitFloat() or tP["isM"]:
+                # FIXME: Don't know why for grvw == 1, need further investigate
```

Comment:
what is this FIXME meaning?

Reply:
The logic is written in data type case by case. It should be written in blockWidth or something.